README.md

```diff
@@ -5,7 +5,7 @@ Analysis of inference methods on standard population models
 We recommend you start by creating a new `conda` environment for the analysis. 
 
 ```
-conda create -n popsim_env_test python=3.6.8 --yes
+conda create -n popsim_env_test --yes -c conda-forge -c terhorst smcpp python=3.7 
```

Comment:
hey @ckyriazis this is the big change in the readme that makes the install go well. give it a try on a new conda env?

Reply:
i see...I'll give it a try! thanks

Reply:
hey @ckyriazis- this working for you? if so let's merge

Reply:
@andrewkern i just tried this and am unable to create the conda environment due to incompatibilities in these packages (the list goes on) 
![image](https://user-images.githubusercontent.com/11757173/70657015-f349ab00-1c0f-11ea-8e4f-1baf7b67952b.png)

Reply:
oh wow this is weird. can you give me more system info?

Reply:
also that first step there is just installing a new environment and smcpp with it-- weird that it would instantly choke there on your system

Reply:
this was a clean install on a mac. i just tried on linux and it appears to have worked, so i'm guessing thats the issue. 

one other thing i noticed though - the pandas=0.24.2 specification i included was due to an issue with the masking code (there was an update in 0.25 that deprecated something that was used here), so wont be covered this installation approach.

Reply:
okay should we roll back that edit and pin the pandas version then?

Reply:
yea lets do that

Reply:
done. i'm *not* going to rebase this one as I think that commit message will be informative in the future when we try to unroll the dependencies